tests/php/Forms/NumericFieldTest.php

```diff
@@ -127,6 +127,16 @@ public function testNumberTypeOnInputHtml()
         $this->assertContains('type="text"', $html, 'number type not set');
     }
 
+    public function testNullSet() {
+        $field = new NumericField('Number');
+        $field->setValue('');
+        $this->assertEquals($field->Value(), null);
```

Comment:
Sorry to be pedantic, but the expected value is on the left, expression on the right. :)